### DISCUSSION TITLE
History Widget 1x2

### DISCUSSION THREAD

**Discussion by pphu** [iSoron/uhabits#1513] [Feature Requests]
Hello,

I would like the History Widget to be able to get as small as 1x2. When getting that narrow, using Java Calendar's [NARROW_FORMAT](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html#NARROW_FORMAT) can save space when showing the weekday names. I started a proof of concept on my local env to see how it looks. See the screenshot: 
![1x2 widget example](https://user-images.githubusercontent.com/6785210/192081376-0807a131-0458-4227-b948-a0547ea839a0.png)

**Comment by pphu** [comment]
I pushed up a PR. https://github.com/iSoron/uhabits/pull/1514

**Reply by fmoham18** [reply]
Did these stop working? I can no longer downsize my 2x2 history widget to 1x2
